pkg/cmd/oauth-apiserver/cmd.go

```diff
@@ -17,6 +17,7 @@ import (
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apiserver/pkg/authorization/union"
 	"k8s.io/apiserver/pkg/endpoints/discovery/aggregated"
+	_ "k8s.io/apiserver/pkg/features"
```

Comment:
I don't think this is necessary. We should already be picking up the feature gates via other apiserver library imports.